### PR TITLE
docs(es): translates feb 22 to feb 27 docs

### PR DESCRIPTION
**Pull request by icarusgk**
Addresses the changes made from Feb-22 to Feb-27.

Some commits were left out as they are already solved in the spanish tranlation:
- [Vue School's link was already in the right place](https://github.com/vuejs/docs/commit/63e0246286fb8b00701fc72e2e7d0977e7b09cc8)
- [The 'and' was already placed](https://github.com/vuejs/docs/commit/eb162a66adf2cb8a1e04bc576a5b0c93c27ad61e)

### CODE REVIEW

File: src/guide/extras/reactivity-transform.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # Transformación de la Reactividad {#reactivity-transform}
 
-:::warning Característica Experimental
-La Transformación de la Reactividad es actualmente una función experimental. Está desactivada por defecto y requiere [opt-in explícito](#opt-in-explicito). También podría cambiar antes de ser finalizada. Para estar al día, sigue su [propuesta y discusión en GitHub](https://github.com/vuejs/rfcs/discussions/369).
+:::danger Característica Experimental Obsoleta
+La Transformación de la Reactividad era una función experimental y ha quedado obsoleta. Por favor, lea sobre [el razonamiento aquí](https://github.com/vuejs/rfcs/discussions/369#discussioncomment-5059028).
+
+Eventualmente será elimidad del núcleo de Vue en una futura versión menor. Si todavía quieres usarlo, ahora está disponible a través del plugin [Vue Macros](https://vue-macros.sxzz.moe/features/reactivity-transform.html).
```
Comment (glopzel) [inline src/guide/extras/reactivity-transform.md]: Eventualmente sera "elimidad" => eliminado

**Review by glopzel — COMMENTED** [review]
Found 1 small error, the rest LGTM

File: src/guide/extras/reactivity-transform.md (hunk 1 above)
Comment (icarusgk) [reply src/guide/extras/reactivity-transform.md]: Lo cambié a "eliminada" por ser "una" caracteristica 😄

**Review by glopzel — APPROVED** [review]
LGTM

**Review by glopzel — APPROVED** [review]

**Pull request merged**